applications/quantumleap/chart/values.yaml

```diff
@@ -0,0 +1,70 @@
+quantumleap:
+  # Default values for quantumleap.
+  # This is a YAML-formatted file.
+  # Declare variables to be passed into your templates.
+  
+  # -- Number of replicas
+  replicaCount: 2
+  image:
+    # -- Container image to deploy
+    repository: smartsdk/quantumleap
```

Comment:
Can set the full image location? sth like docker.io/smartsdk/quantumleap (or wherever this comes from)?

Reply:
Done